state-chain/pallets/cf-lp/src/lib.rs

```diff
@@ -306,6 +314,12 @@ pub mod pallet {
 				Self::free_balances(&account_id)?.iter().all(|(_, amount)| *amount == 0),
 				Error::<T>::FundsRemaining
 			);
+			ensure!(
```

Comment:
we could add a test for this with a slightly smarter mock

Reply:
yeah sure! what did you have in mind?

Reply:
Just that this error is thrown when there are boost funds basically